helpers/skills/notebooks-cve-resolution/SKILL.md

```
@@ -0,0 +1,200 @@
+  --base rhoai-X.Y \
+  --title "RHAIENG-XXXX: Fix CVE-YYYY-NNNNN in package" \
+  --body "..."
+```
+
+**If any step fails:** Log the failure and continue to next CVE.
+
+## Step 5: Comment on Jira Tickets
+
+For each successful fix, use Jira MCP `addCommentToJiraIssue`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)